README.md

```diff
@@ -0,0 +1,33 @@
+# Haiku Inc Website
+
+This repository contains the source code for [Haiku Inc's website](https://www.haiku-inc.org).
+
+## Adding new content
```

Comment:
I don't know if we really need this information in this repo. The inc website is in a separate repository because it is supposed to be seldom modified and only by people who have a good reason to edit it. This looks like information more suited to the readme of just the main website repo.

Reply:
Yeah, that is true, but I thought it might be a good idea to leave a note for whoever comes across this and decides to edit something in future.

Reply:
@waddlesplash I felt it might be a good idea for whoever at Haiku decides to edit the website and finds this repo - there was no documentation for me when I got started moving the pages over from the old site.

Reply:
I think it would be better to have this on the main website repo; the inc repo follows the same structure that does.

Reply:
I think it's fine to have this documentation here, it is not worded as a call for contributors, more as documentation for people who need to do something with the website.